www/vorj00/ukol_07/assets/js/main.js

```diff
@@ -0,0 +1,59 @@
+var points = 0;
+var cities = ['Barcelona', 'Dortmund', 'Madrid', 'Turin', 'Prague', 'New York', 'Paris', 'London', 'Beijing', 'Moscow'];
+var firstSelect = null;
+
+cities = cities.concat(cities);
+cities.sort(function() { return .5 - Math.random(); });
+
+function editPoints(change){
+    if((change > 0) || (points > 0)){
+        points+=change;
+    } else {
+        points = 0;
+    }
+}
+
+$(document).ready(function(){
+    const pexesoPoints = '#js-points';
+    const pexesoBoard = '#js-pexeso';
+    const pexesoCard = '.js-playable';
+
+    $(pexesoPoints).text(points);
+
+    $.each(cities, function(index, value){
+        $(pexesoBoard).append(`<div data-id="${index}" class="pexeso__card pexeso__card--playing js-playable"></div>`)
+    });
+
+    $(pexesoCard).on("click", function(){
+        let currentCard = $(this);
```

Comment:
behem jednoho tahu hrac nesmi otocit vice nez 2 karty, proste bych na zacatek dal podminku jestli jiz jsou dve karty otoceny, tak ukoncit callback. to se pozna napr nejakym globalnim counterem.